components/new-contribution-flow/index.js

```diff
@@ -282,6 +294,25 @@ class ContributionFlow extends React.Component {
       }
     }
 
+    // Check that the contributor is not blocked from contributing to the collective
+    const collectiveBlockedCategories = get(
+      this.props.collective,
+      'settings.moderation.rejectedContributionCategories',
+      [],
+    );
```

Comment:
We should be consistent in our variable names, it's confusing to move from `rejected` to `blocked`. Let's stick to `rejected`

Now that I see it again,`rejectedContributionCategories` is a bit verbose, could be shortened to `rejectedCategories`.

Reply:
This will require a change on the API as well